styles/_colors.scss

```diff
@@ -2,6 +2,7 @@
 $primary-color: #50e39e !default;
 $primary-color-lighten-1: lighten($primary-color, 7%) !default;
 $primary-color-darken-1: darken($primary-color, 7%) !default;
+$primary-color-darken-2: darken($primary-color, 12%) !default;
```

Comment:
I did this on purpose, so we can change % without having breaking change.

1 -> ligther
2 -> even ligher

or

1 -> darker
2 -> even more darker